lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/EvidenceFactory.java

```diff
@@ -67,6 +71,14 @@ public Object[] create(KBVItem kbvItem) throws JsonProcessingException {
         return new Map[] {objectMapper.convertValue(evidence, Map.class)};
     }
 
+    private int getVerificationScoreForPass(EvidenceRequest evidenceRequest) {
+        if (Objects.isNull(evidenceRequest)) {
+            return VC_PASS_EVIDENCE_SCORE;
+        }
+        int verificationScore = evidenceRequest.getVerificationScore();
+        return verificationScore > 0 ? verificationScore : VC_PASS_EVIDENCE_SCORE;
+    }
+
```

Comment:
Let's extract this code into KBV lib util folder, we can have it called by, the new overload this way

```
    public void setVerificationScore(EvidenceRequest evidenceRequest) {
        this.verificationScore = EvidenceUtil.getVerificationScore(evidenceRequest);
    }
```
if we do this we can also use it here  https://github.com/govuk-one-login/ipv-cri-kbv-api/pull/345/files#diff-88f028988d98316335066950a4387f73c4f3c0a7dd782b1420507d88d46d4358R256-R270 when this [PR](https://github.com/govuk-one-login/ipv-cri-lib/pull/504) is merged - see https://govukverify.atlassian.net/browse/OJ-2612